pkgs/top-level/all-packages.nix

```diff
@@ -3366,7 +3366,9 @@ with pkgs;
 
   merriweather-sans = callPackage ../data/fonts/merriweather-sans { };
 
-  meson = callPackage ../development/tools/build-managers/meson { };
+  meson = meson_0_57;
+  meson_0_57 = callPackage ../development/tools/build-managers/meson/0.57 { };
```

Comment:
If we are going to remove this soon, I would not introduce `meson_0_57` attribute.

Reply:
I think it smooths the transition.

When we start some sprintable in the near future, we will find some packages that depend on the older Meson; so, maintaining a `meson_0_57` attribute can be used on these cases as a safe fallback in this transition.

Reply:
Looking at other similar packages, it will likely just cause the old Meson to remain in tree forever. And maintaining the old Meson is precisely what I do not want to do. It is already hard to review the changes (need to checkout the branch and compare with meld).

Reply:
Solved